repos/rust-lang/www.rust-lang.org.toml

```
@@ -0,0 +1,20 @@
+org = "rust-lang"
+name = "www.rust-lang.org"
+description = "The home of the Rust website"
```

To accommodate https://github.com/rust-lang/team/pull/1353.

```suggestion
description = "The home of the Rust website"
homepage = "https://www.rust-lang.org"
```